assets/js/components/BatchEdit/PreviewItems.js

```diff
@@ -32,41 +32,34 @@ const previewItem = css`
   }
 `;
 
-export default function BatchEditPreviewItems({ items = [] }) {
+export default function PreviewItems({ items = [] }) {
   return (
-    <div>
-      <p className="mb-2">
-        <strong>Items preview</strong>
-      </p>
-      <div className="is-centered ">
-        <ul css={inlineList} data-testid="list-preview-items">
-          {items.map(({ id, representativeFileSet }) => (
-            <li
-              key={id}
-              className="mr-4 mb-4 is-inline-block"
-              css={previewItem}
-            >
-              <Link to={`/work/${id}`} target="_blank" className="hvr-shrink">
-                <figure>
-                  <img
-                    data-testid="image-preview"
-                    src={
-                      representativeFileSet.url
-                        ? `${representativeFileSet.url}/square/128,128/0/default.jpg`
-                        : "https://bulma.io/images/placeholders/128x128.png"
-                    }
-                  />
-                </figure>
-              </Link>
-            </li>
-          ))}
-        </ul>
-      </div>
+    <div className="is-centered ">
+      <ul css={inlineList} data-testid="list-preview-items">
+        {items.map(({ id, representativeFileSet, representativeImage }) => (
+          <li key={id} className="mr-4 mb-4 is-inline-block" css={previewItem}>
+            <Link to={`/work/${id}`} target="_blank" className="hvr-shrink">
+              <figure>
+                <img
+                  data-testid="image-preview"
+                  src={
+                    representativeImage
+                      ? `${representativeImage}/square/128,128/0/default.jpg`
+                      : representativeFileSet.url
+                      ? `${representativeFileSet.url}/square/128,128/0/default.jpg`
+                      : "https://bulma.io/images/placeholders/128x128.png"
+                  }
+                />
+              </figure>
+            </Link>
+          </li>
+        ))}
+      </ul>
```

Comment:
How about we extract this component to the `/UI` directory now that we're using it in 2 different places.